src/functions/users/flextime/handler.ts

```diff
@@ -39,40 +37,23 @@ export const listUsersFlextimeHandler: APIGatewayProxyHandler = async (event) =>
             }
           };
         } catch (error) {
-          /**
-           * Per-user error handling
-           */
-          console.error(`Error fetching flextime for user ${severaUser.guid}:`, error);
           return {
```

Comment:
Here you are returning the body content, see the return type in the catch at the bottom of this file. This is fine but you should also return the status code:
Something like:
{
 statusCode: error instanceof Error && (error as any).statusCode ? (error as any).statusCode : 500,
      body: JSON.stringify({
            code: error instanceof Error && (error as any).statusCode ? (error as any).statusCode : 500,
            message: `Failed to fetch flextime for user ${severaUser.guid}`,
            error: error instanceof Error ? error.message : "Unknown error"
      })
    };
    
    Also try to test locally, do we really need such verbose error parsing?